simulator/risc_v/t/unit_test.cpp

```diff
@@ -547,6 +547,48 @@ TEST_CASE("RISCV RV64 ror")
     }
 }
 
+TEST_CASE("RISCV RV32 rori")
+{
+    // |  01100  |     imm     |   rs1   | 101 |    rd   |   0010011   |  RORI
+    CHECK( RISCVInstr<uint32>( 0x6007D793).get_disasm() == "rori $a5, $a5, 0"); // 01100 | 0000000 (0) | 01111 ($a5) | 101 | 01111 ($a5) | 0010011
+    TestData<uint32>test_0 = TestData<uint32>( 0x01234567, 0,  0x01234567); // src2 (0) is not used
+    INFO( "Iteration: " << 0);
+    test_0.make_test ("rori", 0);
+    CHECK( RISCVInstr<uint32>( 0x6047D793).get_disasm() == "rori $a5, $a5, 4"); // 01100 | 0000100 (4) | 01111 ($a5) | 101 | 01111 ($a5) | 0010011
+    TestData<uint32>test_4 = TestData<uint32>( 0x01234567, 0,  0x70123456); // src2 (0) is not used
+    INFO( "Iteration: " << 1);
```

Comment:
This violates [AAA principle](https://github.com/MIPT-ILab/mipt-mips/wiki/Unit-Tests#3a-rule). I propose to move all test cases to separate CATCH test cases. Then, you won't need "INFO()" as error messages would tell you what test failed, and you won't need these "_4" suffixes.